browser/brave_ads/ads_tab_helper.h

```diff
@@ -91,6 +91,8 @@ class AdsTabHelper : public content::WebContentsObserver,
   bool is_browser_active_;
   std::vector<GURL> redirect_chain_;
   bool should_process_;
+  uint32_t html_hash_;
```

Comment:
Maybe need to set initial values for class members?

Reply:
ah yes this was a std::string previously so did not need to, I will fix now